src/gluonts/dataset/repository/_m1.py

```diff
@@ -0,0 +1,102 @@
+import json
+import os
+from pathlib import Path
+from typing import NamedTuple, List, Dict
+from urllib import request
+from zipfile import ZipFile
+
+from gluonts.dataset.repository._util import metadata, to_dict, save_to_file
+
+from ._tsf_reader import TSFReader
+
+ROOT = "https://zenodo.org/record"
+
+dataset_info = {
+    "m1_yearly": {"file": "m1_yearly_dataset.zip", "record": "4656193"},
+    "m1_quarterly": {"file": "m1_quarterly_dataset.zip", "record": "4656154"},
+    "m1_monthly": {"file": "m1_monthly_dataset.zip", "record": "4656159"},
+}
+
+
+def download_dataset(description: str, path: Path):
+    request.urlretrieve(
+        f"{ROOT}/{description['record']}/files/" f"{description['file']}",
+        path / description["file"],
+        )
+
+
+def frequency_converter(freq: str):
```

Comment:
What are the excpected inputs?

Reply:
The frequency from the loaded datasets are written out, e.g. `yearls`, `daily`, `4_seconds`, etc.
That's the reason for the conversion